canonical-web-and-design/ubuntu.com
# Fix cypress tests to ensure they pass

## Done

- Bumped Cypress to latest version
- Set cookies on the test files to prevent the cookie policy from blocking the page
- Fixed up tests in `form_spec.js` and `navigation_spec.js`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- For the post to marketo status 200 to pass you need to add the credentials to a local file named `cypress.env.json`. Please ping me so I can tell you where to find them
- In your terminal run `yarn run cypress open --config-file tests/cypress/cypress.json --config baseUrl=http://0.0.0.0:8001/`
- This will open a cypress window - click on the `form_spec.js` and then the `navigation_spec.js` to see the tests pass.
- Note: if the form field is out of view when it's attempted to be selected it will fail - you need to scroll as the test fills out the fields 


## Issue / Card

Fixes [#4365](https://github.com/canonical-web-and-design/web-squad/issues/4365)
